docs/apis/avalanchego/apis/issuing-api-calls.md

```diff
@@ -16,15 +20,111 @@ where
+
+:::caution
+When setting up RPC access to a remote node, make sure you don't leave the `http-port` accessible to everyone! There are malicious actors that scan for nodes that have unrestricted access to their RPC port and then use those nodes for spamming them with resource-intensive queries which can knock the node offline. Only allow access to your node's RPC port from known IP addresses!
+:::
+
+### Endpoint path
+
+Each API’s documentation specifies what endpoint path a user should make calls to in order to access the API’s methods.
+
+In general, they are formatted like:
+
+```
+/ext/[api-name]
+```
+
+So for the Admin API, the endpoint path is `/ext/admin`, for the Info API it is `/ext/info` and so on. Note that not some APIs have additional path components, most notably the chain RPC endpoints which includes the subnet chain RPCs. We'll go over those in detail in the next section.
+
+So, in combining the base URL and the endpoint path we get the complete URL for making RPC calls. For example, to make a local RPC call on the Info API, the full URL would be:
+
+```
+127.0.0.1:9650/ext/info
+```
+
+## Primary Network and Subnet RPC calls
+
+Besides the APIs that are local to the node, like Admin or Metrics APIs, nodes also expose endpoints for talking to particular chains that are either part of the Primary Network (the X, P and C chains), or part of any Subnets the node might be syncing or validating.
+
+In general, chain endpoints are formatted as:
+
+```
+ext/bc/[blockchainID]
+```
+
+### Primary network endpoints
+
+The Primary Network consists of three chains: X, P and C chain. As those chains are present on every node, there are also convenient aliases defined that can be used instead of the full blockchainIDs. So, the endpoints look like:
+
+```
+ext/bc/X
+ext/bc/P
+ext/bc/C
+```
+
+### C-Chain and Subnet EVM endpoints
+
+C-Chain and many subnets run a version of the EthereumVM (EVM). EVM exposes its own endpoints, which are also accessible on the node: JSON-RPC, and Websocket.
+
+#### JSON-RPC EVM endpoints
+
+To interact with C-Chain EVM via the JSON-RPC use the endpoint:
+
+```
+/ext/bc/C/rpc
+```
+
+To interact with subnet instances of the EVM via the JSON-RPC endpoint:
+
+```
+/ext/bc/[blockchainID]/rpc
+```
+
+where `blockchainID` is the ID of the blockchain running the EVM. So for example, the RPC URL for the Swimmer Network (a subnet that runs the Crabada game) running on a local node would be:
+
+```
+http://127.0.0.1/ext/bc/2K33xS9AyP9oCDiHYKVrHe7F54h2La5D8erpTChaAhdzeSu2RX/rpc
```

Comment:
is http optional or mandatory? I was wondering that we should add http and https in the doc.